frontend/src/components/Events/NewEventTemplateForm.js

```diff
@@ -186,11 +211,12 @@ const NewEventTemplateForm = () => {
                   showTime={{
                     hideDisabledOptions: true
                   }}
-                  format="YYYY-MM-DD HH:mm"
+                  format="DD-MM-YYYY HH:mm"
```

Comment:
Why did you swap the date and year?

Reply:
easy to read in front end and follow DD-MM-YYYY convention